examples/integration/BUILD

```diff
@@ -15,16 +15,51 @@ load(
 
 exports_files(["README.md"])
 
-xcodeproj(
-    name = "xcodeproj",
-    associated_extra_files = ASSOCIATED_EXTRA_FILES,
-    build_mode = "xcode",
-    config = CONFIG,
-    extra_files = EXTRA_FILES,
-    project_name = "Integration",
-    scheme_autogeneration_mode = SCHEME_AUTOGENERATION_MODE,
-    schemes = get_xcode_schemes(),
-    tags = ["manual"],
-    top_level_targets = XCODEPROJ_TARGETS,
-    unfocused_targets = UNFOCUSED_TARGETS,
-)
+# Example of xcodeproj cache warming
+#
+# 1. Register a single macOS execution platform in WORKSPACE
+#    register_execution_platforms("@build_bazel_apple_support//platforms:macos_x86_64")
+# 2. Create a `platform_mappings` file
+# 3. Add `build --@build_bazel_rules_swift//swift:universal_tools` flag to your
+#    .bazelrc
+# 4. Define an xcodeproj target for each macOS platform, e.g.
+#    `xcodeproj-x86_64` and `xcodeproj-sim_arm64`. Even if you don't need to
+#    support any of iOS, tvOS, or watchOS platforms, you still need to set all
+#    of the `ios_simulator_cpus`, `tvos_simulator_cpus`, and
+#    `watchos_simulator_cpus` to avoid Starlark transition effect. See
+#    https://github.com/buildbuddy-io/rules_xcodeproj/issues/1094
+# 5. Run each of the following commands for produce cache for each macOS host
+#    platform. This can be run from an Intel or Apple silicon Mac.
+#
+#    bazel run //:xcodeproj-x86_64 \
+#      --@com_github_buildbuddy_io_rules_xcodeproj//xcodeproj:extra_common_flags=--remote_download_minimal \
+#      -- \
+#      --build_output_groups all_targets
+#
+#    bazel run //:xcodeproj-sim_arm64 \
+#      --@com_github_buildbuddy_io_rules_xcodeproj//xcodeproj:extra_common_flags=--remote_download_minimal \
+#      -- \
+#      --build_output_groups all_targets
```

Comment:
Since this is about to change, and I'll be adding it to `usage.md`, I'm going to wait to merge this PR.